client/components/domains/transfer-domain-step/transfer-restriction-message.jsx

```diff
@@ -112,9 +114,20 @@ class TransferRestrictionMessage extends React.PureComponent {
 								{ reason }
 							</div>
 							<div className="transfer-domain-step__section-action">
-								<Button compact onClick={ this.goToMapDomainStep }>
+								<Button
+									className="transfer-domain-step__section-action-button"
+									compact
+									onClick={ this.goToMapDomainStep }
+								>
 									{ translate( 'Connect domain without transferring' ) }
 								</Button>
+								<Button
+									className="transfer-domain-step__section-action-button"
+									compact
+									onClick={ goBack }
+								>
+									{ translate( 'Transfer different domain' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Transfer a Domain' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).